web/src/app/page.tsx

```diff
@@ -0,0 +1,13 @@
+"use client";
+import IndexPage from "./index-page";
+
+export default async function Page() {
+  return <IndexPage />;
+}
+
+// NOTE: won't work until this is a server page, and it can't be a server page
+// until Chakra fixes support for this new app directory stuff...
+// export const metadata = {
```

Comment:
You could make your server component `page` basically just have metadata, data fetching, and then forward everything to a client component child to do the styling with chakra.

Reply:
Makes sense, thanks!